src/cursor.cpp

```diff
@@ -130,10 +158,11 @@ static void clear_cache()
 			cache[n] = NULL;
 		}
 	}
-
+#if !SDL_VERSION_ATLEAST(2,0,0)
```

Comment:
As far as I can make out, `cursor_buf` is only used in the `draw()` and `undraw()` functions. I removed this section initially because, having removed those functions already, `cursor_buf` was no longer referenced at all other than here (and if the only thing you do with a variable is set it to null if it's not, then obviously you don't need it anymore).